scripts/install.sh

```diff
@@ -1,33 +1,29 @@
 #!/usr/bin/env bash
 set -x
 
-DOCKER_EXPORT='export DOCKER_HOST=tcp://172.17.42.1:2375'
-
 final_setup() {
   sudo bash -c 'echo "127.0.0.1 $(hostname)" 2>/dev/null >> /etc/hosts'
   sudo stop docker
-  sudo bash -c 'echo DOCKER_OPTS=\"-H 0.0.0.0:2375 -g /opt/docker_images \" >> /etc/default/docker'
```

Comment:
Ah damn!
We need the `-H 0.0.0.0:2375` for the `DOCKER HOST` setup, but not for the `DOCKER CLIENT` setup.
It was that way considering we could use the install script for both.

I am ok moving away from that idea and eventually put in place a `docker-machine` script to take care of host setup.